src/map/addPopUp.js

```diff
@@ -3,174 +3,181 @@ import { getCurrentState } from '../store/actions/actions';
+                                // for null values
+                                if (layer.popup && layer.popup.hideNulls) {
+
+                                    content =
+                                        `<div><b>${rowItem[layer.popup.header]}</b></div>` +
+                                        `<div><center>${Mustache.render(layer.popup.body, commaFormatting(layer, datum, true))}</center></div>`;
+                                } else {
+                                    const bodyProperties = layer.popup.body.match(/{{(.*?)\}}/g).map(val => val.replace(/{{?/g, '').replace(/}}?/g, ''));
+                                    bodyProperties.forEach(val => {
+                                        // Check if rowItem[val] is a string and if it has ,
+                                        const contentArr = typeof rowItem[val.trim()] !== 'number' &&
+                                            rowItem[val.trim()].includes(',') ? rowItem[val.trim()].split(',') : [];
+                                        if (contentArr.length > 1) {
+                                            rowItem[val] = contentArr.join(', ');
+                                        }
+                                    });
+                                    content =
+                                        `<div>` +
+                                        `<div><b>${row[layer.popup.header]}</b></div>` +
+                                        `<div><center>${Mustache.render(
+                                            layer.popup.body,
+                                            commaFormatting(layer, rowItem, true)
+                                        )}</center></div>` +
+                                        `</div>`;
+                                }
+                            } else {
+                                content = Mustache.render(layer.popup.body, commaFormatting(layer, (layer.popup && layer.popup.hideNulls) ? datum : rowItem, true));
+                            }
+                            break;
+                        }
+                    }
+                } else {
+                    // if no data, use feature.properties to populate popup
+                    // eslint-disable-next-line no-lonely-if
+                    const formattedData = commaFormatting(layer, feature.properties, true);
+                    if (feature.properties[layer.popup.header]) {
+                        content = `<div><b>${feature.properties[layer.popup.header]}</b></div>`;
+                        if (layer.popup.body) {
+                            content += `<div><center>${Mustache.render(layer.popup.body, formattedData)}</center></div>`;
+                        }
+                    } else {
+                        content = Mustache.render(
+                            layer.popup.body,
+                            formattedData,
+                        );
+                    }
+                }
```

Comment:
@cKellyDesign I branches off from staging and I think prettier is the one formatting my code in this case